CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [3.9.0](https://github.com/marklogic/mlxprs/releases/tag/v3.9.0)
+
+- New MarkLogic status view including high-level information about the currently connected MarkLogic server
+- Can now be used to connect to a MarkLogic server behind a proxy server or in the ML Cloud
+- Updated the user documents to use GitHub Pages.
```

Comment:
Users likely don't know what GH pages is - just include a link to the new docs - e.g. "[Improved user guide.](link goes here)".